.travis.yml

```diff
@@ -19,16 +34,28 @@ deploy:
   # Push image to Dockerhub on merge to master
   - provider: script
     skip_cleanup: true
-    script: docker push nfvpe/sriov-device-plugin
+    script: >
+     bash -c '
+     docker tag nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:$ARCH;
+     docker push nfvpe/sriov-device-plugin:$ARCH;
+     sudo docker manifest create -a nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:amd64 nfvpe/sriov-device-plugin:ppc64le;
+     sudo docker manifest annotate nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:amd64 --os linux --arch amd64;
+     sudo docker manifest annotate nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:ppc64le --os linux --arch ppc64le;
+     sudo docker manifest push nfvpe/sriov-device-plugin -p;
+     echo foo'
     on:
       branch: master
   # Push image to Dockerhub on tag
   - provider: script
     skip_cleanup: true
     script: >
       bash -c '
-      docker tag nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:$TRAVIS_TAG;
-      docker push nfvpe/sriov-device-plugin:$TRAVIS_TAG;
+      docker tag nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:"$TRAVIS_TAG-$ARCH";
+      docker push nfvpe/sriov-device-plugin:"$TRAVIS_TAG-$ARCH";
+      sudo docker manifest create -a nfvpe/sriov-device-plugin:$TRAVIS_TAG nfvpe/sriov-device-plugin:"$TRAVIS_TAG-amd64" nfvpe/sriov-device-plugin:"$TRAVIS_TAG-ppc64le";
```

Comment:
what does ``docker manifest`` do here? do we want to use a experimental command here?

Reply:
It is creating like a map when the user run `docker pull image:latest` it will return the image:amd64 if the machine arch is amd64 or ppc64le if it is the arch. of the machine